yeonjae/programmers-2/점프와순간이동.js

```diff
@@ -0,0 +1,19 @@
+function solution(n) {
+  let answer = 0;
+
+  while (n > 0) {
+    if (n % 2 !== 0) {
+      n--;
+      answer++;
+    } else {
+      n = n / 2;
+    }
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(n) {
  let answer = 0;

  while (n > 0) {
    if (n % 2 !== 0) {
      n--;
      answer++;
    } else {
      n /= 2;
    }
  }
  return answer;
}
```
- 풀이 잘봤숩니다~
- n = n / 2 를 n /= 2 로 단축해서 사용할 수 있습니다 👍